flixel/addons/display/FlxZoomCamera.hx

```diff
@@ -29,7 +29,7 @@ class FlxZoomCamera extends FlxCamera
 	 */
 	private var _zoomMargin:Float;
 	
-	public function new(X:Int, Y:Int, Width:Int, Height:Int, Zoom:Float = 0)
+	public function new(X:Int, Y:Int, Width:Int, Height:Int, Zoom:Float = 0, ZoomSpeed:Float = 25, ZoomMargin:Float = 0.25)
```

Comment:
Oops! I was copying from my local file and forgot to add that change in. I do feel at least the zoom speed is pretty useful to have there.